src/panels/config/entities/ha-config-entities.ts

```diff
@@ -466,6 +512,7 @@ export class HaConfigEntities extends SubscribeMixin(LitElement) {
       (entity) => entity.entity_id === entryId
     );
     if (!entry) {
+      fireEvent(this, "hass-more-info", { entityId: entryId });
```

Comment:
I think that we should open the entity registry dialog, tell what's up instead of showing controls. Related tab is still useful.